### ISSUE TITLE
DOTCOM-25386 - Tab order does not match reading order on blogs homepage

### ISSUE THREAD

**Issue by avanishgandhi** [adobe/theblog#297]
JIRA - https://jira.corp.adobe.com/browse/DOTCOM-25386?src=confmacro

**Context** 
Chrome Version 83.0.4103.116 on macOS. MacBook Pro running macOS Catalina 10.15.5

**URL**: https://blog.adobe.com/

**Reproduction Steps** 

Open http://blog.adobe.com/ and tab through the page using keyboard.
When a keyboard user tabs through the page, they go through the news and articles cards first and then the featured card.
Featured card is placed below the news and article cards in the markup. 
Expected behavior

Tab order should always be same as the reading order. So, place the featured card at the top in the markup to reflect the way it is visually presented. Once done, keyboard users will be able to navigate through the page according to reading order.

 

**WCAG Success Criteria**
Keyboard

**Comment by davidnuescheler** [comment]
should be fixed on staging.

https://staging--theblog--adobe.hlx.page/

**Comment by avanishgandhi** [comment]
Works. Thanks
